packages/date-picker/src/vaadin-date-picker-mixin.js

```diff
@@ -447,11 +461,11 @@ export const DatePickerMixin = (subclass) =>
       super._onBlur(event);
 
       if (!this.opened) {
-        this.__commitParsedOrFocusedDate();
+        const didValueCommitOccur = this.__commitParsedOrFocusedDate();
```

Comment:
**note:** We need to check whether an actual value commit occurred to prevent potential duplicate validation.